homeassistant/components/myuplink/helpers.py

```diff
@@ -49,4 +49,63 @@ def skip_entity(model: str, device_point: DevicePoint) -> bool:
         ):
             return False
         return True
+    if "F730" in model:
+        if any(
+            d in device_point.parameter_name.lower()
+            for d in (
+                "monday",
+                "tuesday",
+                "wednesday",
+                "thursday",
+                "friday",
+                "saturday",
+                "sunday",
+            )
+        ):
+            return True
+        if device_point.parameter_id in [
```

Comment:
Perhaps better to explain in a comment. If this feature will appear in more heat pump models in the future, it would make sense to break this code segment out.

Reply:
I mean, it's a static list, if we can make this a set or tuple outside of the for loop we are more efficient. And I think we should build code for what we know now, not for what we might expect in the future (in the case of an external party like myuplink)

Reply:
Can we also do the same to the parameters?